webapp/src/main/webapp/js/webconferencing.js

```diff
@@ -1632,7 +1632,6 @@
 							$userAction.data("callbuttonuser", userId);
 							// cleanup after previous user
 							$userAction.find(".callButtonContainer").empty();
-							addPopoverButton($userAction, userContext(userId));
```

Comment:
Is the function addPopoverButton ? shouldn't clean the code from it too

Reply:
This function is used in space popover too

Reply:
Do we need it in space popover?

Reply:
I think yes

Reply:
Please ask for this to make the task completed . We do not have the possibility to do group calls , I do not find it logic to have a call button in spaces

Reply:
In case of Spaces, we have only Chat button, and for normal users, we will have both buttons Call and Chat.
Group calls is not available yet.

Reply:
Ok Thanx @hghorbel, I'll put back on Platform 
cc @ahamdi

Reply:
Well does this PR have a sense ? I see no file changes now :thinking:

Reply:
i just added platform_ui dependencie to web-conferencing to use color variables ==> see last commit